pixart.js

```diff
@@ -1 +1,37 @@
 
+$(document).ready(function(){
+  //don't reload the page
+  event.preventDefault();
```

Comment:
You only need to use `event.PreventDefault()` in the event listener for your button. It's the button's default behavior that causes the page to reload. No need to place it here (or in the functions below) -- just the event listener.